articles/native-platforms/ios-swift.md

```diff
@@ -177,6 +177,10 @@ Then add Lock Facebook's Pod
 pod 'Lock-Facebook', '~> 2.0'
 ```
 
+Click on the `Pods` project and select `Lock-Facebook` target and set `Allow Non-modular Includes In Framework Modules` to Yes:
```

Comment:
Not sure this is something I'd rather recommend cause there are cases this will break everything and we shouldn't change the `Pods` project. I know there is an [issue](https://developers.facebook.com/bugs/850078471740674) with FB SDK but haven't tried the solution they proposed.